lib/active_merchant/billing/network_tokenization_credit_card.rb

```diff
@@ -0,0 +1,16 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class NetworkTokenizationCreditCard < CreditCard
```

Comment:
It's essentially a credit card though, it shares all the attributes of a credit card, it just has the extra verification fields